src/native/libs/System.Security.Cryptography.Native/openssl.c

```diff
@@ -1304,6 +1365,9 @@ static int32_t EnsureOpenSsl10Initialized()
     // Ensure that the error message table is loaded.
     ERR_load_crypto_strings();
 
+    // In OpenSSL 1.0.2-, CRYPTO_EX_INDEX_X509 is 10.
+    g_x509_ocsp_index = CRYPTO_get_ex_new_index(10, 0, NULL, NULL, ExDataDup, ExDataFree);
```

Comment:
Looks like there was a const-ness change in the function signature that I'll have to account for. :frown: